cnv/cnv_node_network/cnv-node-network-state.adoc

```diff
@@ -0,0 +1,13 @@
+[id="cnv-node-network-state"]
+= Node network state
```

Comment:
In the menu, we have "Node network state" and "Updating node network configuration". Can we make it more clear that one is for observing and one for configuration? "Observing node network state" would work